### ISSUE TITLE
Inject environment variables

### ISSUE THREAD

**Issue by rd-xx** [rust-build/rust-build.action#74]
As the title says, I want to know if there is a way to inject environment variables into the created Docker container?

At first, I thought that I could export the env var from the pre-build script, as it is executed inside the container, but it doesn't work and I think it might be because of [this](https://unix.stackexchange.com/questions/30189/how-can-i-make-environment-variables-exported-in-a-shell-script-stick-around).

I also might be looking into the wrong thing, I'd appreciate if you could look at my [workflow](https://github.com/rd-xx/color-swapper/blob/main/.github/workflows/build.yml) and my [pre-build script](https://github.com/rd-xx/color-swapper/blob/main/.github/scripts/pre-build.sh).

Thanks

**Comment by Douile** [comment]
You can set environment variables by adding the env property to the job in the github actions workflow. The only time this wouldn't work is if the script overrides the variables you set. 

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idenv

**Comment by rd-xx** [comment]
I did try setting them via the env property in the job, but they were not available inside the container, sorry for not making this clear. However, are you saying this should be the case?

**Comment by Douile** [comment]
Yes, all the action settings used to be (and are still able to be) read via environment variables from within the container. 

https://github.com/rust-build/rust-build.action/blob/master/entrypoint.sh#L20

**Comment by rd-xx** [comment]
After a bit testing, I confirm that I do have access to environment variables from the Docker container. I guess I got confused when I tried it the other day. Thanks :)
